README.md

```diff
@@ -13,8 +12,8 @@ and [Highmaps](http://www.highcharts.com/products/highmaps) component for [Ember
 ## Requirements
 
 * Ember CLI
-* Ember >= 2.12.0
-  * If you need support for Ember < 2.12.0, use ember-highcharts < v1.0.0
+* Ember >= 2.18.0
```

Comment:
Is it worth removing support for Ember 2.X in this releasee as well? It's very old at this point?

Reply:
good call - i went back and forth on this and even ended up getting `ember-2.12` back under the ember:try config; will revert the readme!